lib/modules/manager/tekton/extract.ts

```diff
@@ -80,6 +85,80 @@ function getDeps(doc: TektonResource): PackageDependency[] {
   return deps;
 }
 
+function addPipelineAsCodeAnnotations(
+  annotations:
+    | {
+        [key: string]: string;
+      }
+    | undefined,
+  deps: PackageDependency[],
+): void {
+  if (is.nullOrUndefined(annotations)) {
+    return;
+  }
+
+  for (const annotationsKey in annotations) {
+    if (
+      !annotationsKey.toLowerCase().includes('pipelinesascode.tekton.dev/task')
```

Comment:
why to lowercase?

Reply:
Annotations in kubernetes can both be [uppercase and lowercase](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set), wanted to generalize it so it supports both cases

Reply:
It does not: https://github.com/openshift-pipelines/pipelines-as-code/blob/c0e0e5dd1007413407ce8bfd1d81a2449646a65f/pkg/matcher/annotation_tasks_install.go#L177

Will update to support the same regexp as in PAC